frame/election-provider-support/src/onchain.rs

```diff
@@ -55,77 +55,109 @@ impl From<sp_npos_elections::Error> for Error {
 /// It is advisable to use the former ([`ElectionProvider::elect`]) only at genesis, or for testing,
 /// the latter [`InstantElectionProvider::instant_elect`] for onchain operations, with thoughtful
 /// bounds.
-pub struct OnChainSequentialPhragmen<T: Config>(PhantomData<T>);
-
-/// Configuration trait of [`OnChainSequentialPhragmen`].
 ///
-/// Note that this is similar to a pallet traits, but [`OnChainSequentialPhragmen`] is not a pallet.
+/// Please use `BoundedOnchainExecution` at all times except at genesis or for testing,
```

Comment:
doc above needs updating.

Reply:
Should I change it more? Or you think what I have is good?